packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3632,6 +3632,83 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe(`#beRightBack`, () => {
+        const fakeMultistreamRoapMediaConnection = {
+          createReceiveSlot: sinon.stub().resolves({on: sinon.stub()}),
+          createSendSlot: sinon.stub().returns({
+            setSourceStateOverride: sinon.stub().resolves(),
+            clearSourceStateOverride: sinon.stub().resolves(),
+          }),
+        };
+
+        beforeEach(() => {
+          meeting.meetingRequest.sendBrb = sinon.stub().resolves({body: 'test'});
```

Comment:
I had a discussion with @antsukanova about the stubbing process, using `leaveMeeting` as a reference for testing. It appears that `locusMediaRequestStub` is used for actions related to adding/updating media and publishing streams. In contrast, `locusDeltaRequest` is for member updates. While we can verify member updates through `locusDeltaRequest`, using `sendBrb` seems like a better approach, as it aligns with how other tests were conducted for the functions in `request.ts`.